### PR TITLE
fix: missing type definition file on npm

### PR DESCRIPTION
**Pull request by azu**
<del>Just add `"types"` field to `package.json`.</del>

Add `index.d.ts` to `files` field.

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

### CODE REVIEW

**Comment by loilo** [comment]
Hey there,
thanks for the PR. Can you explain why this is needed? I learned about the `"types"` field a while ago, but in all of my experience tools just pick up a type declaration file if it has the same name as the `.js` file.

Is that not true for every tool?

File: package.json [hunk 1]
```diff
@@ -17,9 +17,11 @@
   "author": "Florian Reuschel <florian@loilo.de>",
   "files": [
     "index.js",
+    "index.d.ts",
```
Comment (azu) [inline package.json:20]: Add `index.d.ts` to distribution files.

Comment (loilo) [reply package.json:20]: Welp, good catch. 😅

**Comment by azu** [comment]
Hi,
I have tested the package, but type checked is failed.
Now, I noticed that this pacakge does not includes `index.d.ts`.

So, `types` field is not related with this problem.
Should I remove `types` field?
Probably, TypeScript detect same file name like `index.js` and `index.d.ts` correctly without `types` field

**Comment by azu** [comment]
>  Should I remove `types` field?

Removed.
This PR just add `index.d.ts` to `files` field.

**Comment by loilo** [comment]
Oops, fat-fingered on the "close" button, sorry.

Alright, thanks for the catch, going to release this change in a moment. 👍

**Comment by loilo** [comment]
Alright, v1.0.2 is out with the type declarations on board. Thanks!

**Pull request merged**